crates/cranelift/src/debug/transform/attr.rs

```diff
@@ -295,6 +295,7 @@ where
                 pending_di_refs.insert(current_scope_id, attr.name(), offset);
                 continue;
             }
+            AttributeValue::String(d) => write::AttributeValue::String(d.to_slice()?.to_vec()),
```

Comment:
I think you can use `into_owned()` instead of `to_vec()`.

Reply:
Thanks, shall  I change `AttributeValue::Block(d) => write::AttributeValue::Block(d.to_slice()?.to_vec()),` as well?

Reply:
I went ahead and changed `Block`, I have limited experience of `Cow` and slicing, but it seemed logical being as how similar `Block` is to `String`